src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -222,18 +229,174 @@ export default function SearchableIntegrationsList({
+					/>
+					<Button
+						disabled={!atLeastOneFacetSelected}
+						text="Reset Filters"
+						icon={<IconX16 />}
+						color="tertiary"
+						onClick={handleClearFilters}
+						className={s.row_fill}
+					/>
+				</div>
+
+				<div className={s.mobileFilters}>
+					<MobileFilters />
+				</div>
+			</Dialog>
 		</div>
 	)
 }
 
+// Renders Tier/Component/Flags checkboxes
+function MobileFilters() {
+	const {
+		tierOptions,
+		officialChecked,
+		setOfficialChecked,
+		partnerChecked,
+		setPartnerChecked,
+		communityChecked,
+		setCommunityChecked,
+		sortedComponents,
+		componentCheckedArray,
+		setComponentCheckedArray,
+		flags,
+		flagsCheckedArray,
+		setFlagsCheckedArray,
+	} = useIntegrationsSearchContext()
+
+	const makeToggleTierHandler = (tier: Tier) => () => {
+		switch (tier) {
+			case Tier.OFFICIAL:
+				setOfficialChecked((s) => !s)
+				break
+			case Tier.PARTNER:
+				setPartnerChecked((s) => !s)
+				break
+			case Tier.COMMUNITY:
+				setCommunityChecked((s) => !s)
+				break
+		}
+	}
+
+	const makeToggleComponentHandler = (index: number) => () => {
+		const newComponentCheckedArray = [...componentCheckedArray]
+		newComponentCheckedArray[index] = !newComponentCheckedArray[index]
+		setComponentCheckedArray(newComponentCheckedArray)
+	}
+
+	const makeToggleFlagHandler = (index: number) => () => {
+		const newFlagsCheckedArray = [...flagsCheckedArray]
+		newFlagsCheckedArray[index] = !newFlagsCheckedArray[index]
+		setFlagsCheckedArray(newFlagsCheckedArray)
+	}
+
+	return (
+		<>
+			<div className={s.optionsContainer}>
+				<Legend>Tier</Legend>
+				{tierOptions.map((e) => {
+					const checked =
+						(e === Tier.OFFICIAL && officialChecked) ||
+						(e === Tier.PARTNER && partnerChecked) ||
+						(e === Tier.COMMUNITY && communityChecked)
+					return (
+						<CheckboxField
+							key={e}
+							labelFontWeight="regular"
+							label={capitalize(e)}
+							checked={checked}
+							onChange={makeToggleTierHandler(e)}
+						/>
+					)
+				})}
+			</div>
+
+			<div className={s.optionsContainer}>
+				<Legend>Component</Legend>
+				{sortedComponents.map((e, i) => {
+					const checked = componentCheckedArray[i]
+					return (
+						<CheckboxField
+							key={e.id}
+							labelFontWeight="regular"
+							label={capitalize(e.plural_name)}
+							checked={checked}
+							onChange={makeToggleComponentHandler(i)}
+						/>
+					)
+				})}
+			</div>
+
+			<div className={s.optionsContainer}>
+				<Legend>Flags</Legend>
+				{flags.map((e, i) => {
+					const checked = flagsCheckedArray[i]
+					return (
+						<CheckboxField
+							key={e.id}
+							labelFontWeight="regular"
+							label={e.name}
+							checked={checked}
+							onChange={makeToggleFlagHandler(i)}
+						/>
+					)
+				})}
+			</div>
+		</>
+	)
+}
+
 function FilterBar({ filterQuery, onChange }) {
 	return (
 		<div className={s.searchBar}>
 			<IconSearch16 />
+			{/* TODO: consider <FilterInput/> */}
```

Comment:
👀  (this is good to keep here if it's something for later, just want to call this out that it's sill here if you intended to take care of this here).

Reply:
Yea, keeping this for _sometime after this PR_